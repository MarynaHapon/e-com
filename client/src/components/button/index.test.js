// Core
import React from 'react';
import { shallow } from 'enzyme';

// Component
import { Button } from './index';

it('expect to render Button component', () => {
  expect(shallow(
    <Button variant='primary' onClick={() => {}}>
      Go to checkout
    </Button>
  )).toMatchSnapshot();
});
