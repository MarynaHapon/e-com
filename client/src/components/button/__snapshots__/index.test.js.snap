// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Button component 1`] = `
<styled.button
  onClick={[Function]}
  variant="primary"
>
  Go to checkout
</styled.button>
`;
